addons/pos_restaurant/static/src/app/split_bill_screen/split_bill_screen.xml

```diff
@@ -24,12 +24,21 @@
                     </div>
                     <div class="controls flex-column flex-nowrap w-100 w-lg-50">
                         <div class="order-info mb-2 mt-2 mt-lg-0 py-3 py-lg-4 rounded-3 text-center bg-view">
+                            <div class="fs-4 text-start ms-2" style="font-size: 0.33em;">
```

Comment:
the bootstrap `fs-*` class should control the font size; why do you need the css ?